flink-connector-pulsar/src/main/java/org/apache/flink/connector/pulsar/sink/PulsarSinkOptions.java

```diff
@@ -137,12 +137,24 @@ private PulsarSinkOptions() {
                     .withDescription(
                             Description.builder()
                                     .text(
-                                            "The metrics from Pulsar Producer are only exposed if you enable this option.")
+                                            "The metrics from Pulsar Producer are only exposed if you enable this option. ")
                                     .text(
                                             "You should set the %s to a positive value if you enable this option.",
                                             code(PULSAR_STATS_INTERVAL_SECONDS.key()))
                                     .build());
 
+    public static final ConfigOption<Boolean> PULSAR_VALIDATE_SINK_MESSAGE_BYTES =
+            ConfigOptions.key(SINK_CONFIG_PREFIX + "validateSinkMessageBytes")
```

Comment:
@syhily What's your plan if we support more sink message schema validation beyond `Schema.AUTO_PRODUCE_BYTES`?

Reply:
Well. I notice that other schemas are already configured in `createMessageBuilder` if non-null. This PR is for AUTO_PRODUCE_BYTES only and no other schema validations remains unhandled?

Reply:
The schema validation is disabled by default. We add the validation only if the user want to enable it for bytes messages. Otherwise, they should enable schema evolution check.